course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -83,6 +84,7 @@
                         roleName: $scope.getRoleName(user.role_id),
                     });
                 });
+            $scope.searchInProgress = false;
```

Comment:
for this, are you thinking use the $q for the post and the get? like you have on line 258 
```           $q.all([peoplePromise, memberPromise])
                .then($scope.handleLookupResults);```